data/json/items/comestibles/frozen.json

```diff
@@ -0,0 +1,155 @@
+    "nutrition": 55,
+    "quench": 9,
+    "fun": 25,
+    "health": -6,
+    "price": 400,
+    "material": [ "milk", "junk" ]
+  },
+  {
+    "type": "COMESTIBLE",
+    "id": "icecream",
+    "name": "ice cream",
+    "name_plural": "ice cream scoops",
+    "description": "A sweet, frozen food made of milk with liberal amounts of sugar.",
+    "weight": 270,
+    "volume": 4,
+    "container": "plastic_bucket",
+    "comestible_type": "FOOD",
+    "symbol": "~",
+    "color": "white",
+    "spoils_in": 24,
+    "nutrition": 15,
+    "fun": 15,
+    "healthy": -1,
+    "price": 250,
+    "material": "milk",
+    "phase": "liquid",
+    "//": "not technically a drink, but is quasi-liquid enough that it'd get all over everything if you carried it",
+    "flags": [ "MELTS" ]
+  },
+  {
+    "type": "COMESTIBLE",
+    "copy-from": "icecream",
+    "id": "icecream_artificial",
+    "name": "dairy dessert",
+    "name_plural": "dairy dessert scoops",
+    "description": "Government regulations dictate that since this isn't *technically* ice cream, it be called a dairy dessert instead.  It still tastes good, but your body won't like you.",
+    "healthy": -2,
+    "price": 200,
+    "material": [ "milk", "junk" ]
+  },
+  {
+    "type": "COMESTIBLE",
+    "copy-from": "icecream",
+    "id": "icecream_candy",
+    "name": "candy ice cream",
+    "name_plural": "candy ice cream scoops",
+    "description": "Ice cream with bits of chocolate, caramel, or other flavoring mixed in.",
+    "fun": 20,
+    "healthy": -3,
+    "price": 300,
+    "material": [ "milk", "junk" ]
+  },
+  {
+    "type": "COMESTIBLE",
+    "copy-from": "icecream",
+    "id": "icecream_fruit",
+    "name": "fruity ice cream",
+    "name_plural": "fruity ice cream scoops",
+    "description": "Small bits of sweet fruit have been tossed into this ice cream, making it slightly less terrible for you.",
+    "healthy": 0,
+    "nutrition": 18,
+    "fun": 17,
+    "price": 300,
+    "material": [ "milk", "fruit" ]
+  },
+  {
+    "type": "COMESTIBLE",
+    "copy-from": "icecream",
+    "id": "icecream_custard",
+    "name": "frozen custard",
+    "name_plural": "frozen custard scoops",
+    "description": "Similar to ice cream, this treat made famous in Coney Island is made like ice cream, but with egg yolk added in.  Its storing temperature is warmer, and it lasts a little longer than regular ice cream.",
+    "spoils_in": 48,
+    "nutrition": 17,
+    "price": 300,
+    "material": [ "milk", "egg" ]
+  },
+  {
+    "type": "COMESTIBLE",
+    "copy-from": "icecream",
+    "id": "icecream_yogurt",
+    "name": "dessert yogurt",
+    "name_plural": "dessert yogurt",
+    "description": "Tarter than ice cream, this is made with yogurt and other dairy products, and is generally low-fat compared to ice cream itself.",
+    "nutrition": 14,
+    "price": 275,
+    "material": [ "milk" ]
+  },
+  {
+    "type": "COMESTIBLE",
+    "copy-from": "icecream",
```

Comment:
As sorbet copy-from `icecream` it still has a `milk` as material despite stating it is made from water with sweetener.